index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <!--<![endif]-->
+<html>
+
```

Comment:
Kindly do not leave unused commands as comments. It makes the code not to be clean.

Reply:
Nice job being polite, and giving reasons for the requested change :+1:, however I suggest you also phrase your comments as suggestions instead of orders

Example:
:x: Wrong
Kindly do not leave unused commands as comments. It makes the code not to be clean.

:heavy_check_mark: Right
I kindly suggest you do not leave unused commands as comments. It makes the code not to be clean.